File: lib/models/ProductModel.ts
import mongoose, { Schema } from "mongoose";

export interface IProduct {
    id: string;
    name: string;
    price: number;
    description?: string;
    image?: string;
}

const productSchema = new Schema<IProduct>({
    name: {
        type: String,
        required: true,
    },
    price: {
        type: Number,
        required: true,
    },
    description: {
        type: String,
    },
    image: {
        type: String,
    },
});

const Product =
    mongoose.models.Product || mongoose.model("MyDb", productSchema);

export default Product;

File: Baim_API/Baim_API/Models/Product/ProductModel.cs
using DB.Models;

namespace Baim_API.Models.Product;
public class ProductModel
{
	public int Id { get; set; }
	public string Id1C { get; set; } = string.Empty;
	public string Name { get; set; } = string.Empty;
	public string Description { get; set; } = string.Empty;
	public string? Image { get; set; }
	public bool IsPublic { get; set; } = false;
	public string ProductType { get; set; } = string.Empty; 
}